demo/index.html

```diff
@@ -831,27 +831,35 @@
               compact: false,
               pageSizeOptions: [1, 10, 50],
               pageSize: 2,
-              feed: hx.dataTable.objectFeed({
-                headers: [
-                  { name: 'Name', id: 'name', groups: ['Child Group 1', 'Parent Group'] },
-                  { name: 'Age', id: 'age', groups: ['Child Group 1', 'Parent Group'] },
-                  { name: 'Profession', id: 'profession', groups: ['Child Group 2', 'Parent Group'] }
-                ],
-                rows: [
-                  {
-                    id: 0, // hidden details can go here (not in the cells object)
-                    cells: { 'name': 'Bob', 'age': 25, 'profession': 'Developer' }
-                  },
-                  {
-                    id: 1,
-                    cells: { 'name': 'Jan', 'age': 41, 'profession': 'Artist' }
-                  },
-                  {
-                    id: 2,
-                    cells: { 'name': 'Dan', 'age': 41, 'profession': 'Builder' }
-                  }
-                ]
-              })
+              feed: {
```

Comment:
Could we also keep the object feed example in as well? - so have both this one and the old version, just to make any manual testing a bit easier